vcpkg/maintainers/functions/vcpkg_msbuild_install.md

```diff
@@ -0,0 +1,121 @@
+---
+title: vcpkg_msbuild_install
+description: Learn how to use vcpkg_msbuild_install.
+ms.date: 08/14/2023
+---
+
+# vcpkg_msbuild_install
+
+Build and install a msbuild-based project. This replaces `vcpkg_install_msbuild` and all other msbuild related functions.
+`vcpkg_msbuild_install` will generate a props and target file containing the cmake toolchain build flags and include it via `/p:ForceImportAfterCpp(Props|Targets)`
```

Comment:
```suggestion
`vcpkg_msbuild_install` will generate a props file and target file containing the CMake toolchain build flags and include it via `/p:ForceImportAfterCpp(Props|Targets)`
```

Needs links to explain what target and props files are.

Reply:
This is more intended to inform the user that `/p:ForceImportAfterCpp(Props|Targets)` is an internally used property.